lib/licensed/source/pip.rb

```diff
@@ -32,13 +42,15 @@ def dependencies
         end
       end
 
+      private
+
       # Build the list of packages from a 'requirements.txt'
-      # Assumes that the requirements.txt follow the format pkg=1.0.0 or pkg==1.0.0
       def parse_requirements_txt
-        File.open(@config.pwd.join("requirements.txt")).map do |line|
-          p_split = line.split("=")
-          p_split[0]
-        end
+        File.open(@config.pwd.join("requirements.txt")).map { |line|
```

Comment:
I chose the first `{ }` because of [chaining](https://github.com/rubocop-hq/ruby-style-guide#single-line-blocks), and the second `{ }` because it's a single-line statement but multi-lined for readability when using split panes in an editor.

I can switch to `do end` if you feel strongly about this. Note that the rubocop checks passed - this could be an opportunity to configure the rules to be stricter.

Reply:
Yeah, I'd prefer that you change the multiline to `do end`.  I know `do end` isn't terribly pretty with chaining but to be fair neither is `{ }` when multiline 😆 .

Reply:
Resolved in f0afebc16d3757354ef2508abf774dd45197e2d5